doc/source/using-as-a-library.rst

```diff
@@ -157,16 +157,16 @@ Any exceptions that occur during the execution of the automagic will be returned
 Run the plugin
 --------------
 
-Firstly, we should check whether the plugin will be able to run (ie, whether the configuration options it needs
-have been successfully set).  We do this as follow (where plugin_config_path is the base_config_path (which defaults
+Firstly, we should check whether the plugin will be able to run (i.e., whether the configuration options it needs
+have been successfully set).  We do this as follows, where plugin_config_path is the base_config_path (which defaults
 to `plugins` and then the name of the class itself):
 
 ::
 
     unsatisfied = plugin.unsatisfied(context, plugin_config_path)
 
 If unsatisfied is an empty list, then the plugin has been given everything it requires.  If not, it will be a
-Dictionary of the hierarchy paths and their associated requirements that weren't satisfied.
+dictionary of the hierarchy paths and their associated requirements that weren't satisfied.
```

Comment:
dictionary is a standard python class, so to be specific it should be `dict` rather than dictionary?